charts/azure-service-operator/values.yaml

```diff
@@ -25,6 +25,18 @@ azureUseMI: False
 # azureSecretNamingVersion allows choosing the algorithm used to derive secret names. Version 2 is recommended.
 azureSecretNamingVersion: "2"
 
+# purgeDeletedKeyVaultSecrets determines if the operator should issue a secret Purge request in addition
+# to Delete when deleting secrets in Azure Key Vault. This only applies to secrets that are stored in Azure Key Vault.
+# It does nothing if the secret is stored in Kubernetes.
+purgeDeletedKeyVaultSecrets: False
```

Comment:
They're not mutually exclusive, you could set both to true if you say, wanted to purge on delete but recover on failure in case somehow something wasn't purged.

Do you think they _should_ be mutually exclusive? They could be combined into a `softDeleteHandling` or something, that was either `Purge|Recover|None`